### PR TITLE
return path onDone

### PR DESCRIPTION
**Pull request by RZulfikri**
it was update form PR https://github.com/prscX/react-native-photo-editor/pull/11 .
I have test PR https://github.com/prscX/react-native-photo-editor/pull/11. IOS works fine but android not return any path, so I update few code. Now Android and IOS will return temporary path after editing.

thanks to @tonimoeckel 

can someone test this cc @prscX , I thing everyone need this.. 😆

### CODE REVIEW

**Comment by cdan1995** [comment]
@prscX  Can you merge pull requests this feature?

File: android/src/main/java/ui/photoeditor/RNPhotoEditorModule.java [hunk 1]
```diff
@@ -20,7 +19,7 @@
 
 public class RNPhotoEditorModule extends ReactContextBaseJavaModule {
 
-  private static final int PHOTO_EDITOR_REQUEST = 1;
+  private static final int PHOTO_EDITOR_REQUEST = 1539266202;
```
Comment (naoey) [inline android/src/main/java/ui/photoeditor/RNPhotoEditorModule.java]: i believe this is causing an error to be thrown. the message is "Can only use lower 16 bits for requestCode".

Comment (RZulfikri) [reply android/src/main/java/ui/photoeditor/RNPhotoEditorModule.java]: sorry my bad. revert to 1.

Thanks.

**Comment by prscX** [comment]
Thanks a lot @RZulfikri for all the Effort you have added to this PR.

I have merged [PR: 54](https://github.com/prscX/react-native-photo-editor/pull/54) which also contains these PR changes. Considering that, I am closing this PR.

Thanks
</ Pranav >

**Pull request closed without merging**